apollo-router/src/plugin/test/mock/subgraph.rs

```diff
@@ -101,6 +101,7 @@ impl Service<SubgraphRequest> for MockSubgraph {
                     "couldn't find mock for query {}",
                     serde_json::to_string(&req.subgraph_request.body()).unwrap()
                 ))
+                .extension_code("FETCH_ERROR".to_string())
```

Comment:
maybe make the common extension code constants?

Reply:
I tempted to say we don't care because the only places we use it is in tests and mock